test/support/mocks.ex

```diff
@@ -11,6 +11,7 @@ Mox.defmock(Dotcom.Redix.PubSub.Mock, for: Dotcom.Redix.PubSub.Behaviour)
 
 Mox.defmock(CMS.Api.Mock, for: CMS.Api.Behaviour)
 Mox.defmock(MBTA.Api.Mock, for: MBTA.Api.Behaviour)
+Mox.defmock(Stops.Repo.Mock, for: Stops.Repo.Behaviour)
```

Comment:
This should be moved to the Repo block below

Reply:
I knew I forgot something when chugging through the conflict resolutions!! This was that something :D